node/src/actors/rad_manager/handlers.rs

```diff
@@ -12,6 +13,8 @@ use crate::actors::messages::{ResolveRA, RunTally};
 use super::RadManager;
 use futures::FutureExt;
 
+const MAX_RETRIEVAL_TIMEOUT: Duration = Duration::from_millis(10000);
```

Comment:
We should document this: why it is needed, why this default value.

Reply:
Also add that this must be lower than half an epoch, and the commit transactions must have enough time to propagate through the entire network before block mining begins.